main-packages-list.txt

```diff
@@ -4,6 +4,7 @@ gdisk
 genisoimage
 httpd
 httpd-tools
+inotify-tools
```

Comment:
I suggest we remove the inotify-related code, we don't follow that code path where it's used.

Reply:
makes sense